core/chain_manager.go

```diff
@@ -731,20 +731,26 @@ func (self *ChainManager) InsertChain(chain types.Blocks) (int, error) {
 	return 0, nil
 }
 
-// diff takes two blocks, an old chain and a new chain and will reconstruct the blocks and inserts them
-// to be part of the new canonical chain.
-func (self *ChainManager) diff(oldBlock, newBlock *types.Block) (types.Blocks, error) {
+// reorgs takes two blocks, an old chain and a new chain and will reconstruct the blocks and inserts them
```

Comment:
English nitpicking: this mixes [simple future tense](http://www.edufind.com/english-grammar/simple-future-tense/) and [simple present tense](http://www.edufind.com/english-grammar/simple-present-tense/)

Would recommend to always go with present tense for source code comments describing code after the comment and past tense for comments describing code prior to it.